emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/DynamoDBClient.java

```diff
@@ -292,7 +292,12 @@ public BatchWriteItemResult call() throws
 
               double consumed = 0.0;
               for (ConsumedCapacity consumedCapacity : result.getConsumedCapacity()) {
-                consumed = consumedCapacity.getCapacityUnits();
+                consumed = consumedCapacity.getTable().getCapacityUnits();
+                if(null!=consumedCapacity.getLocalSecondaryIndexes()) {
+                  for (String name : consumedCapacity.getLocalSecondaryIndexes().keySet()) {
```

Comment:
Might be better to iterate `.values() ` collection to get the capacity units directly

Reply:
done